hasjob/models/jobpost.py

```diff
@@ -434,6 +433,10 @@ def viewcounts_key(jobpost_id):
             return ['hasjob/viewcounts/%d' % post_id for post_id in jobpost_id]
         return 'hasjob/viewcounts/%d' % jobpost_id
 
+    @property
+    def max_counts_key():
+        return u'maxcounts'
```

Comment:
This is a static value. It doesn't need to be a property, and it doesn't even need to be on the model. Make this a constant in the relevant view module.